node/src/BaseClient.ts

```diff
@@ -6874,12 +6877,11 @@ export class BaseClient {
      * ```
      */
     public async pubsubNumSub(
-        channels: string[],
-        options?: DecoderOption,
+        options?: { channels?: GlideString[] } & DecoderOption,
     ): Promise<{ channel: GlideString; numSub: number }[]> {
         return this.createWritePromise<GlideRecord<number>>(
-            createPubSubNumSub(channels),
-            options,
+            createPubSubNumSub(options?.channels),
+            { decoder: options?.decoder },
```

Comment:
Anyway you can simplify that
```suggestion
            options,
```